airflow-core/docs/administration-and-deployment/logging-monitoring/logging-tasks.rst

```diff
@@ -172,7 +172,7 @@ Serving logs from workers and triggerer
 
 Most task handlers send logs upon completion of a task. In order to view logs in real time, Airflow starts an HTTP server to serve the logs in the following cases:
 
-- If ``SequentialExecutor`` or ``LocalExecutor`` is used, then when ``airflow scheduler`` is running.
+- If ``LocalExecutor`` is used, then when ``airflow scheduler`` is running.
 - If ``CeleryExecutor`` is used, then when ``airflow worker`` is running.
```

Comment:
not related to this PR but what does the sentence even mean:

maybe improve it in a follow-up

Reply:
Oh lol, let me update in a follow-up :)